pkg/pgmodel/pgx_test.go

```diff
@@ -608,19 +631,15 @@ func TestPGXInserterInsertData(t *testing.T) {
 				}
 			}
 
-			for i, metric := range mock.CopyFromTableName {
-				name := metric[1]
-				for metric, tableName := range mockMetrics.metricCache {
-					if tableName == name {
-						name = metric
-					}
-				}
-				result := c.rows[name]
+			// for i, metric := range mock.CopyFromTableName {
```

Comment:
do we want to do any check with mock.Times etc here?

Reply:
nah, the data is never populated anyway